packages/app-project/src/components/AuthModal/components/RegisterForm/RegisterForm.js

```diff
@@ -0,0 +1,235 @@
+                <TextInput
+                  disabled={isSubmitting}
+                  id={emailFieldId}
+                  name='email'
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  placeholder={counterpart('RegisterForm.emailPlaceholder')}
+                  type='email'
+                  value={values.email}
+                />
+              </FormField>
+
+              <FormField
+                error={errors.realName && touched.realName && errors.realName}
+                help={counterpart('RegisterForm.realNameHelp')}
+                htmlFor={realNameFieldId}
+                label={<Label>{counterpart('RegisterForm.realName')}</Label>}
+              >
+                <TextInput
+                  disabled={isSubmitting}
+                  id={realNameFieldId}
+                  name='realName'
+                  onChange={handleChange}
+                  onBlur={handleBlur}
+                  placeholder={counterpart('RegisterForm.realNamePlaceholder')}
+                  type='text'
+                  value={values.realName}
+                />
+              </FormField>
+
+              <FormField
+                error={errors.privacyAgreement && touched.privacyAgreement && errors.privacyAgreement}
+                htmlFor={privacyAgreementFieldId}
+                required
+              >
+                <CheckBox
+                  checked={values.privacyAgreement}
+                  disabled={isSubmitting}
+                  id={privacyAgreementFieldId}
+                  label={<Label>{counterpart('RegisterForm.privacyAgreement')}{' '}<PrivacyPolicyLink /></Label>}
+                  onChange={handleChange}
+                />
+              </FormField>
+
+              <FormField
+                error={errors.emailListSignUp && touched.emailListSignUp && errors.emailListSignUp}
+                htmlFor={emailListSignUpFieldId}
+              >
+                <CheckBox
+                  checked={values.emailListSignUp}
+                  disabled={isSubmitting}
+                  id={emailListSignUpFieldId}
+                  label={<Label>{counterpart('RegisterForm.emailListSignUp')}</Label>}
+                  onChange={handleChange}
+                />
+              </FormField>
+
+              <FormField
+                error={errors.betaListSignUp && touched.betaListSignUp && errors.betaListSignUp}
+                htmlFor={betaListSignUpFieldId}
+              >
+                <CheckBox
+                  checked={values.betaListSignUp}
+                  disabled={isSubmitting}
+                  id={betaListSignUpFieldId}
+                  label={<Label>{counterpart('RegisterForm.betaListSignUp')}</Label>}
+                  onChange={handleChange}
+                />
+              </FormField>
+
+              <FormField
+                error={errors.underageWithParent && touched.underageWithParent && errors.underageWithParent}
```

Comment:
there won't be any errors for a non-required boolean field - not sure on the legalese, but maybe this should actually be "i am over 16, or if under 16 have completed this with a parent" and we make it required

Reply:
This is using the current text on the registration form, which I'm assuming was reviewed by lawyers(?) at Oxford. @camallen can confirm this I think.

Reply:
please copy the text verbatim from the PFE login forms, changes would need review again.

Reply:
The current registration form also adds fields (and text?) when the under-16 box is ticked. We’ll have to copy that behaviour across too, with the same wording.